airflow/www/static/feed.js

```diff
@@ -0,0 +1,57 @@
+// TODO: Move this script and the RSS icon asset to the dpadmin repo
```

Comment:
Need to add license header to pass license check.

Reply:
@milton0825 I copy-pasted a license header from another file, but it's still showing up in the list of files missing the header. What am I missing?